spec/services/form1010cg/service_spec.rb

```diff
@@ -14,7 +14,6 @@
         },
         'ssnOrTin' => Faker::IDNumber.valid.remove('-'),
         'dateOfBirth' => Faker::Date.between(from: 100.years.ago, to: 18.years.ago).to_s,
-        'gender' => %w[M F].sample,
```

Comment:
so this should have no gender key by default?

Reply:
I think this is true since gender is no longer required

Reply:
Yes, gender should not be present in the minimalist example.